diff --git a/Lib/GetCaretPosEx/GetCaretPosEx.ahk b/Lib/GetCaretPosEx/GetCaretPosEx.ahk
index ff9a7f7..b3ee558 100644
--- a/Lib/GetCaretPosEx/GetCaretPosEx.ahk
+++ b/Lib/GetCaretPosEx/GetCaretPosEx.ahk
@@ -332,8 +332,12 @@ end:
     }
 
     static getWindowScale(hwnd) {
-        if winDpi := DllCall("GetDpiForWindow", "ptr", hwnd, "uint")
-            return A_ScreenDPI / winDpi
+        try {
+            if winDpi := DllCall("GetDpiForWindow", "ptr", hwnd, "uint")
+                return A_ScreenDPI / winDpi
+        } catch {
+            ; ignore error
+        }
         return 1
     }
 
